jbot/src/main/java/me/ramswaroop/jbot/core/facebook/FbService.java

```diff
@@ -11,12 +10,13 @@
  */
 @Service
 public class FbService {
+    private final RestTemplate restTemplate;
+    private final FbApiEndpoints fbApiEndpoints;
 
-    @Autowired
-    RestTemplate restTemplate;
-
-    @Autowired
-    FbApiEndpoints fbApiEndpoints;
+    public FbService(RestTemplate restTemplate, FbApiEndpoints fbApiEndpoints) {
```

Comment:
Don't we need an `@Autowired` annotation here (just above the constructor)?

Reply:
No, @Autowired isn't necessary for constructor injection

Reply:
yea, I saw my IDE put a bean icon just beside it after I posted my comment above. Great, thanks!